biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -1294,11 +1299,29 @@ private void extractFromJar(Jar jar, String source, String destination, boolean
 
 			error("Can not find JAR file '%s'", source);
 		} else {
+
+			Function<String, String> f = v -> v;
```

Comment:
Perhaps we should use a more descriptive variable name than `f`.  Maybe `nameMapper`?